cryostatio/cryostat-helm
# feat(eventtemplates): declarative configuration of Event Template ConfigMaps

Fixes #214
Related to https://github.com/cryostatio/cryostat-helm/issues/225
Depends on https://github.com/cryostatio/cryostat/pull/726

To test:
1. Create a configmap containing a JFR event template: `oc create configmap mytemplate --from-file=mytemplate.jfc`
2. `helm install --set core.image.repository=quay.io/andrewazores/cryostat --set core.image.tag=event-templates-declarative-1 cryostat --set core.config.eventTemplates.configMapNames='{mytemplate}' ./charts/cryostat`
3. Follow the post-install port forwarding steps, then open the Cryostat Web UI. Define a `localhost:0` custom target. Go to Events, then Event Templates, and ensure that the `mytemplate` custom Event Template is listed.
